### PR TITLE
Drop grepping for id tags

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

File: docs/user/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+Bug fixes
+^^^^^^^^^
+
+* Fix updateinfo <id> tag detection (:issue:`3261`).
+
+
+Contributors
+^^^^^^^^^^^^
+
+The following developers contributed to Bodhi 4.0.1:
```
Comment (cverna) [inline docs/user/release_notes.rst]:
```suggestion
The following developers contributed to Bodhi 4.0.2:
```

File: docs/user/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+v4.0.2
+------
+
+This is a bugfix release.
+
+
+Bug fixes
+^^^^^^^^^
+
+* Fix updateinfo <id> tag detection (:issue:`3261`).
```
Comment (bowlofeggs) [inline docs/user/release_notes.rst]:
```suggestion
* Drop updateinfo <id> tag detection (:issue:`3269`).
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

**Comment by bowlofeggs** [comment]
Maybe we should change the commit message to "Drop grepping for <id> tags" instead of "Fix", since we are actually dropping it.

File: docs/user/release_notes.rst (hunk 2 above)
Comment (bowlofeggs) [reply docs/user/release_notes.rst]: And Maybe we should s/Fix/Drop/ here too.

**Review by bowlofeggs — APPROVED** [review]
Thanks!

**Pull request merged**